docs/source/tutorial/00-introduction.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "0. Introduction"
+---
+
+Welcome! This tutorial demonstrates adding the Apollo Android SDK to an app to communicate with a GraphQL server. In this tutorial you will learn how to:
+
+* Download as schema.
+* Write queries, mutations and generate the corresponding Kotlin models.
```

Comment:
```suggestion
* Write queries, mutations to generate the corresponding Kotlin models.
```

Reply:
^ minor disagreement on English structure here - probably a better change if you want it is `Write queries and mutations to generate the corresponding Kotlin models`.

Reply:
Agreed. That's better